man/deepsea-monitor.1

```diff
@@ -0,0 +1,66 @@
+.TH deepsea-monitor 1 "DeepSea CLI man page"
+.SH NAME
+deepsea-monitor - Starts the stage execution monitor.
+
+.SH SYNOPSIS
+deepsea monitor [--help] [--show-dynamic-steps] [--show-state-steps]
+                [--simple-output]
+
+.SH DESCRIPTION
+Starts DeepSea progress monitor.
+
+This allows to visualize DeepSea execution progress when running stages
```

Comment:
"The progress monitor provides a detailed, real-time visualization of what is happening during execution of stages" ?